src/main/java/uk/gov/hmcts/reform/iacasedocumentsapi/domain/utils/AsylumCaseUtils.java

```diff
@@ -149,4 +150,14 @@ public static double getFeeRemission(AsylumCase asylumCase) {
             return getAmountRemitted(asylumCase);
         }
     }
+
+    public static boolean isValidUserDirection(
```

Comment:
Just a question, is it covered by tests?

Reply:
copied from notifications api and no tests exist for this method